src/blocks/TableOfContents/style.scss

```
@@ -0,0 +1,157 @@
+					color: var(--toc-link-color, inherit);
+					display: inline-block;
+					transition: color 0.2s ease;
+
+					&:hover {
+						color: var(--toc-link-hover-color, currentColor);
+						text-decoration: underline;
+					}
+
+					&.active {
+						font-weight: 600;
+						border-left: 3px solid currentColor;
```

Stylelint (17.11.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)